src/styles/blocks/card.scss

```diff
@@ -0,0 +1,80 @@
+@import 'variables';
+
+.card {
+  background-color: $white;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
```

Comment:
No need to hardcode the height for the card
```suggestion
```

Reply:
+++